added a pr template for github (#1086)
popcorndao/sweet-caramel: .github/ISSUE_TEMPLATE/pr_template.md

## Describe your changes

## Issue ticket number

*Please also add this PR to the corresponding ticket.*

## Checklist before requesting a review

[ ] I have crossed off all todos in the issue ticket.

[ ] There are no more TODOS in the PR.

> [ ] There are no more consol.logs for debuging or other temporary functions and comments.

Reviewer: ```suggestion
[ ] There are no more console.logs for debugging or other temporary functions and comments.
```

> [ ] Functiones and variables have clear, distinct names.

Reviewer: ```suggestion
[ ] Functions and variables have clear, distinct names.
```

## Describe your changes

## Issue ticket number
*Please also add this PR to the corresponding ticket.*

## Checklist before requesting a review
*Remove the Smart Contract or Frontend if its not applicable for your issue*

- [ ] I have crossed off all todos in the issue ticket.
- [ ] There are no more TODOS in the PR.
- [ ] There are no more console.logs for debugging or other temporary functions and comments.
- [ ] Functions and variables have clear, distinct names.
- [ ] I performed a self review of the code.
- [ ] All tests are passing.
- [ ] The build and ci checks are successful.

> For Smart Contracs:

Reviewer: ```suggestion

For Smart Contracts:
```

- [ ] We have 100% code coverage.
- [ ] Deploy scripts are included and updated.
- [ ] Fuzz tests are implemented when applicable.
- [ ] Forge tests are added/updated.

> - [ ] Al functions have natspec comments.

Reviewer: ```suggestion
- [ ] All functions have natspec comments.
```

> For Frontend Issues:

Reviewer: ```suggestion

For frontend Issues:
```
